src/types/libraries.ts

```diff
@@ -0,0 +1,12 @@
+import {LANGUAGES} from 'constants/libraries';
+
+type LanguageKeyType = keyof typeof LANGUAGES;
```

Comment:
This is not needed if u use `enum`

Reply:
Just before my last commit, I had it the way you are telling. Don't know why I felt this was better. :/ Let me do a force push and revert.